src/extension.ts

```diff
@@ -486,16 +483,16 @@ function initStatusBar() {
 //
 // @parameter command string The command to run
 function maybeRunKubernetesCommandForActiveWindow(command, progressMessage) {
-    let text, proc;
+    let text;
```

Comment:
Suggest we supply a type declaration while we're in here.

Reply:
👍